db/migrate/20230904121544_create_journal_entries.rb

```diff
@@ -0,0 +1,16 @@
+class CreateJournalEntries < ActiveRecord::Migration[7.0]
+  def up
+    create_table :journal_entries do |t|
+      t.integer :journal_id, null: false, index: true, foreign_key: true
+      t.string :title, null: false, unique: true
+      t.text :content, null: false
+      t.text :goal
+
+      t.timestamps
+    end 
+  end
+
+  def down
+    drop_table :journal_entries
+  end
```

Comment:
You change also just keep one method like:

```suggestion
  def change
    create_table :journal_entries do |t|
      t.integer :journal_id, null: false, index: true, foreign_key: true
      t.string :title, null: false, unique: true
      t.text :content, null: false
      t.text :goal

      t.timestamps
    end 
  end
```

and it will work for both migrating up or down.

Reply:
Laila suggested to do it like this